app/src/main/AndroidManifest.xml

```diff
@@ -19,7 +19,7 @@
             </intent-filter>
         </activity>
         <activity android:name=".SecondActivity" />
```

Comment:
Think we should rename SecondActivity to what it actually does but I don't actually remember what it does XD